File: 22.07 - 15.08 4 wave/Programming examine/2.py
import sys

total = 0

def calculate():
    try:
        line = sys.stdin.readline().strip()
        a, b, c = map(float, line.split())
        d = b**2 - 4*a*c

        if d < 0:
            return
        elif d == 0:
            print(-b / (2*a))
        else:
            print((d**0.5 - b) / (2*a))
            print((-d**0.5 - b) / (2*a))
    except ValueError:
        pass

calculate()

try:
    sys.stdin.readline()  # Keep reading input
except EOFError:
    print(total)
    sys.exit(0)

File: 22.07 - 15.08 4 wave/Programming examine/2.js
var readline = require('readline');
var rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

let total = 0;

process.stdin.on('end', () => { console.log(total); process.exit(0); });

rl.on('line', function (data) {
  let [a, b, c] = data.split(' ').map(a => +a);
  d = Math.pow(b, 2) - 4 * a * c;

  if (d < 0) {
    return
  } else if (d === 0) {
    console.log(-b / (2 * a));
  } else {
    console.log((Math.sqrt(d) - b) / (2 * a));
    console.log((-Math.sqrt(d) - b) / (2 * a));
  }

});